### ISSUE TITLE
Unclear how to return to home screen from a video

### ISSUE THREAD

**Issue by lottebelice** [linkedtv/editortool#18]
When I select a video from the list on the home screen, I don't know how to return to the SV home screen. When I click 'Home', I go to the public home screen, whereas I would except to go to the SV home screen.

**Comment by nicorbb** [comment]
I support this request

**Comment by jblom** [comment]
yes a good point indeed. Just did't get around to properly fix this.
